### ISSUE TITLE
Incorrect text/plain parts exported by org-mime-*-htmlize

### ISSUE THREAD

**Issue by rgemulla** [org-mime/org-mime#32]
Both ```org-mime-org-subtree-htmlize``` and ```org-mime-org-buffer-htmlize``` produce incorrect ```text/plain``` parts on my system (Emacs 26.1, org 9.2.2).

When using ```org-mime-org-subtree-htmlize```, the created ```text/plain``` parts contain a spurious export block. For example:

```
<#multipart type=alternative><#part type=text/plain>#+BEGIN_EXPORT org
... actual text ...
#+END_EXPORT
<#multipart type=related><#part type=text/html><div id="table-of-contents"> ...
```

When exporting with ```org-mime-org-buffer-htmlize```, the text part is missing. For example

```
<#multipart type=alternative><#part type=text/plain><#multipart type=related><#part type=text/html><div id="table-of-contents">...
```

**Comment by redguardtoo** [comment]
`#+BEGIN_EXPORT` looks OK to me. Both Emacs 24 and 26 get same output. It's imported from commit 10098b001c3926

**Comment by redguardtoo** [comment]
817744f remove plain text body (Chen Bin)

I removed embedded text mail. It makes things complicated. `org-mime` need focus on html mail only

**Comment by rgemulla** [comment]
I liked having a plain text part in the export, not everybody reads HTML mail.

**Comment by redguardtoo** [comment]
Could give me a use case why text mail is necessary? I know some mailing list prefer text mail. Then you can use Gnus without org-mime.

**Comment by rgemulla** [comment]
Having the text part allows recipients that use a text-based mail client (such as Emacs in a terminal) to read the mail.

**Comment by redguardtoo** [comment]
I see. Emacs24 already used `shr` to render html mail. It just works out of box, even in terminal.

Cli mail client like mutt does support html mail (with w3m's help). So don't worry too much about text mail.

I know html mails are not welcomed in some mailing list. Then org-mime should be used from the beginning.

**Comment by svictor9** [comment]
A caveat of not having the plain text in the export is that when org-mime-htmlize is applied to a region, that region will now simply not appear to a plain text reader. Before, it would appear in its plain text formating. 

Some people set their mail reader to prefer the plain text version when available. It’s the default of mu4e for instance. To avoid losing message contents on those recipients, org-mime-htmlize should now be applied either to the whole buffer (then the reader will fall back to html for the whole message) or not at all.

**Comment by redguardtoo** [comment]
OK, maybe I should restore plain text output by default, but could be turned off.

**Comment by madalu** [comment]
Yes, please do restore plain text output. The main reason I use org-mime-htmlize is to create multipart/alternative messages consisting of plain text and html. I believe it's fairly standard behavior (and good email etiquette) to create plain text alternatives. For instance, people who use the default settings of mu4e or mutt will simply not be able to see anything in the email.

**Comment by redguardtoo** [comment]
f8eac8c restore plain text mail (Chen Bin)
